x-pack/test/functional/apps/infra/hosts_view.ts

```diff
@@ -247,7 +247,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
 
     // FLAKY: https://github.com/elastic/kibana/issues/159368
     // FLAKY: https://github.com/elastic/kibana/issues/159369
```

Comment:
I think we're removing this in another PR, right?

Reply:
Yes, I removed them here as well :)